### PR TITLE
fix: cannot convert `details`, `summary` block using `useDefaultHTMLSanitizer: false` option

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* fixed that cannot convert `details`, `summary` block using `useDefaultHTMLSanitizer: false` option.

**as-is**

**to-be**

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jung-han — APPROVED** [review]
완료입니다!

**Review by jungeun-cho — APPROVED** [review]
확인했습니다.

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: apps/editor/src/js/mdPreview.js [hunk 1]
```diff
@@ -166,14 +166,14 @@ class MarkdownPreview extends Preview {
...
-        while (el !== endEl) {
+        while (el && el !== endEl) {
           const nextEl = el.nextElementSibling;
 
           el.parentNode.removeChild(el);
           removeOffsetInfoByNode(el);
           el = nextEl;
         }
-        if (el.parentNode) {
+        if (el && el.parentNode) {
```
Comment (shiren) [inline apps/editor/src/js/mdPreview.js:176]: el?.parentNode

File: libs/toastmark/src/__sample__/index.ts [hunk 2]
```diff
@@ -57,12 +57,14 @@ cm.on('change', (editor, changeObj) => {
...
         let el: Element = startEl;
-        while (el !== endEl) {
+        while (el && el !== endEl) {
           const nextEl: Element | null = el.nextElementSibling;
           el.remove();
           el = nextEl!;
         }
-        el.remove();
+        if (el) {
+          el.remove();
```
Comment (shiren) [inline libs/toastmark/src/__sample__/index.ts:66]: el?.remove(); 이렇게 가능하지 않아요?

**Review by shiren — APPROVED** [review]
수고하셨습니다.

**Pull request merged**